packages/astro/src/cli/add/index.ts

```diff
@@ -49,26 +49,44 @@ const ALIASES = new Map([
 	['solid', 'solid-js'],
 	['tailwindcss', 'tailwind'],
 ]);
-const ASTRO_CONFIG_STUB = `import { defineConfig } from 'astro/config';\n\nexport default defineConfig({});`;
-const TAILWIND_CONFIG_STUB = `/** @type {import('tailwindcss').Config} */
+
+const STUBS = {
+	ASTRO_CONFIG: `import { defineConfig } from 'astro/config';\n// https://astro.build/config\nexport default defineConfig({});`,
+	TAILWIND_CONFIG: `/** @type {import('tailwindcss').Config} */
 export default {
 	content: ['./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}'],
 	theme: {
 		extend: {},
 	},
 	plugins: [],
-}\n`;
-const SVELTE_CONFIG_STUB = `\
+}\n`,
+	SVELTE_CONFIG: `\
 import { vitePreprocess } from '@astrojs/svelte';
 
 export default {
 	preprocess: vitePreprocess(),
-};
-`;
-const LIT_NPMRC_STUB = `\
+}\n`,
+	LIT_NPMRC: `\
 # Lit libraries are required to be hoisted due to dependency issues.
 public-hoist-pattern[]=*lit*
-`;
+`,
+	DB_CONFIG: `\
+import { defineDB } from 'astro:db';
+
+// https://astro.build/db/config
```

Comment:
TIL we have these redirects! Agreed we can hold off until a docs preview is ready

Reply:
They _will be added_ by https://github.com/withastro/astro.build/pull/1000! Just need the actual URLs we'll be using.